src/RestClient/endpoints/V3/Themes/ThemeActions.ts

```diff
@@ -0,0 +1,34 @@
+import { AxiosInstance } from 'axios';
+
+import { themesPath } from './Themes';
+import type { ThemeAction } from './types';
+
+class ThemeActions {
+  private client: AxiosInstance;
+
+  constructor(client: AxiosInstance) {
+    this.client = client;
+  }
+
+  /**
+   * Dowloads a stores Theme.
+   *
+   * @param uuid valid store theme UUID
+   * @returns Promsie resolving to a single theme
+   */
+  get(uuid: number): ThemeAction['GetResponse'] {
```

Comment:
a uuid is a string, not a number, right? is this a "number" typ that allows strings? seems confusing.

Reply:
Updated- type to string